docs/user_guide/installation/installation.md

```diff
@@ -0,0 +1,31 @@
+# Using the Deadline Cloud submitter installer
+
+You can install the Deadline Cloud for Blender submitter using the Deadline Cloud submitter installer.
+
+**To install the submitter**
+
+1. Download the [Deadline Cloud submitter installer](https://docs.aws.amazon.com/deadline-cloud/latest/userguide/submitter.html).
+1. Run the installer and follow the prompts.
+1. Launch Blender.
+
+The Deadline Cloud submitter add-on should be automatically enabled.
+> The submitter installer is available for Windows, MacOS, and Linux. See the [developer README](https://github.com/aws-deadline/deadline-cloud-for-blender/blob/mainline/README.md) for manual installation instructions.
+
+**To verify the submitter is installed correctly**
```

Comment:
Can someone verify it's installed by looking for the option in the Render menu? These steps feel a bit more like debugging steps. Could be renamed something like, "Troubleshooting".